protocol/extend.go

```diff
@@ -21,13 +23,13 @@ func (m *Message) EndsTransaction() bool {
 // extended query flow.
 type transaction struct {
```

Comment:
more relevant to previous pr, butr can you please rename `CreatesTransaction` to `CreateTransaction` (i.e. remove the `s`)?

samer for EndsTransaction :)

Reply:
both methods were removed in favor of switch case for the message type